opennms-services/src/main/java/org/opennms/netmgt/poller/pollables/PollableService.java

```diff
@@ -445,11 +446,15 @@ public void schedule() {
         m_schedule.schedule();
     }
 
-    /**
-     * <p>sendDeleteEvent</p>
-     */
-    public void sendDeleteEvent() {
-        getContext().sendEvent(getContext().createEvent(EventConstants.DELETE_SERVICE_EVENT_UEI, getNodeId(), getAddress(), getSvcName(), new Date(), getStatus().getReason()));
+    public void sendDeleteEvent(final boolean ignoreUnmanaged) {
+        final Event event = getContext().createEvent(EventConstants.DELETE_SERVICE_EVENT_UEI, getNodeId(), getAddress(), getSvcName(), new Date(), getStatus().getReason());
+        if (ignoreUnmanaged) {
+            final Parm parm = new Parm();
+            parm.setParmName("ignoreUnmanaged");
```

Comment:
Could use a constant for this too. `org.opennms.netmgt.events.api.EventConstants` would be a good place to put it.